.aicoe-ci.yaml

```diff
@@ -2,7 +2,7 @@ check:
   - thoth-precommit
   - thoth-build
 build:
-  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py36-postgres:latest
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38-postgres:latest
```

Comment:
@fridex @harshad16 should I make this change?

Reply:
We don't have this image built. @harshad16 I don't remember where the Dockerfile was placed for it.

Reply:
we manage the image Dockerfile in the repository, will get the py38 image setup as well.

Reply:
Cool, thanks :+1: